ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -329,11 +300,12 @@ - (void) validatedAcquireTokenByRefreshToken:(NSString*)refreshToken
          
          //Always ensure that the cache item has all of these set, especially in the broad token case, where the passed item
          //may have empty "resource" property:
-         resultItem.resource = _resource;
+         
+         // TODO: add scopes to result
```

Comment:
Definitely need to have scopes in the result here, otherwise the cache will get confused.